website/pages/docs/security/security-models/core.mdx

```diff
@@ -0,0 +1,380 @@
+---
+layout: docs
+page_title: Consul Core Security Model
+sidebar_title: Core
+description: >-
+  Security model including requirements, reccomendations, and threats for the core Consul product.
+---
+
+## Overview 
+
+Consul enables automation of network configurations, service discovery, and secure network connectivity across any 
+cloud or runtime.
+
+Consul utilizes a lightweight gossip and RPC system which provides various essential features. Both of these systems 
+provide security mechanisms which should be utilized to enable confidentiality, integrity and authentication.
+
+Using defense in depth is crucial for Consul security, and deployment requirements may differ drastically depending on 
+your use case. Some security features for multi-tenant deployments are offered exclusively in the enterprise version. 
```

Comment:
```suggestion
your use case. Some security features for multi-tenant deployments are offered exclusively in the Enterprise version. 
```
Should this also link to the enterprise doc?

Reply:
Link added!